src/fmgc/src/components/EfisLabels.ts

```diff
@@ -0,0 +1,32 @@
+import { FlightLevel } from '@fmgc/guidance/vnav/verticalFlightPlan/VerticalFlightPlan';
+import { FmgcComponent } from '@fmgc/lib/FmgcComponent';
+import { FlightPlanManager } from '@fmgc/wtsdk';
+
+export class EfisLabels implements FmgcComponent {
+    private lastTransitionAltitude: Feet;
+
+    private lastTransitionLevel: FlightLevel;
+
+    private flightPlanManager: FlightPlanManager;
+
+    init(): void {
+        // FIXME we need a better way to fetch this... maybe just make it a singleton
+        this.flightPlanManager = FlightPlanManager.DEBUG_INSTANCE;
+    }
+
+    update(_deltaTime: number): void {
```

Comment:
**thought**: Variable is unused.
**thought**: Variable is prefixed with `_` for no reason.
**suggestion**: Remove the unused variable.

Reply:
Our lint rules require the prefix. Since this is a function defined by the super class I prefer to keep the signature the same, even if not required by JS. What do you think?

Reply:
An unused argument apparently is required to have a `_` prefix. Supposedly because a argument named `_` is sometimes used to pattern match any value, or ignore the value. A problem I see in various spots within our code base is the `_variableName` pattern staying around when the argument _is_ used. For example, `FmsMessages.ts` defines a used `_deltaTime`.

Personally I'd still remove the argument when unused, because there's nothing in the type system that forces you to keep the argument. If however you feel strongly about keeping it. I'd rename it to `_`.